CHANGELOG.md

```diff
@@ -33,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   did not have any files associated.
 - Ensured that digests are always added to built RPMs. Previously they would not be included unless
   the "signature-meta" (or "signature-pgp") features were enabled.
+- Move to opt-in for reproducability inhibiting by default values, `build_time` and `build_host`.
```

Comment:
The way this reads makes it kind of ambiguous, "reproducability inhibiting" and "by default" need hyphens.  But rewriting it might be better:

```suggestion
- To facilitate reproducible builds, stop writing `build_time` to the package by default.  Users can configure it with `RPMBuilder::build_time()`.
```

We've never had a release with `build_host` so no point mentioning that.